.gitignore

```diff
@@ -1,2 +1,4 @@
 *~
-*.zip
\ No newline at end of file
+*.zip
+dist
+node_modules
```

Comment:
I would prefer using a trailing slash in got ignore to refer to directories when we know they are directories.

```
node_modules/
```